src/servers/http_server.cc

```diff
@@ -361,6 +392,10 @@ HTTPAPIServer::ResponseRelease(
   LOG_VERBOSE(1) << "HTTP release: "
                  << "size " << byte_size << ", addr " << buffer;
 
+  // hack to delete evbuffer pair structure during release
+  auto response_pair = reinterpret_cast<evbuffer_pair*>(buffer_userp);
+  delete response_pair;
```

Comment:
ResponseRelease will be called for each output buffer... so I don't think this is correct. You will get double-free errors I believe.